src/se/mod.rs

```diff
@@ -366,7 +366,17 @@ impl<'r, W: Write> Serializer<'r, W> {
 
     /// Configure indent for a serializer
     pub fn indent(&mut self, indent_char: char, indent_size: usize) -> &mut Self {
-        self.ser.indent = Indent::Owned(Indentation::new(indent_char as u8, indent_size));
+        self.indent_with_len(indent_char, indent_size, 0)
+    }
+
+    /// Set initial indent level for a serializer
+    pub fn indent_with_len(
```

Comment:
Sounds fine to me.  Perhaps these functions ought to be annotated as `pub(crate)` just to make their intended use clear (even if the module structure prevents them from being public API)?

Reply:
I mean, that `Serializer::indent` will be a public API in next (0.27.0) version, in 0.26.0 that method does not exist. Because of that, we can change it. On the other hand, configuring initial indent is not required as frequent as configuring indent size and character.

Reply:
>On the other hand, configuring initial indent is not required as frequent as configuring indent size and character.

What would be the use case for something like that, perhaps pre-generating bits of XML that can be appended back together later?